readme.md

```diff
@@ -31,6 +33,22 @@ As a computer expert, we want to be more efficient and do our jobs better. We kn
 1. [remove a folder](#removefolder)
 1. [list folder contents](#listfolder)
 
+## <a id="findFile"></a>find a stale file
+
+Find all files modified more than 5 days ago
+
+```bash
+find . -mtime +5
+```
+
+## <a id="showCal"></a>show a calendar
+
+Display a text calendar, along with the week numbers
+
+```
+cal -w
```

Comment:
That's so weird, but I think the -w not working in Debian is something wrong since 2010... but, according to the man page, it should work...